.gitignore

```diff
@@ -1,4 +1,5 @@
 .DS_Store
+npm-debug.log
```

Comment:
We may want to leave this out. You should have a clean build before submitting a PR.